m2cgen/assemblers/boosting.py

```diff
@@ -9,23 +10,17 @@ class BaseBoostingAssembler(ModelAssembler):
 
     classifier_name = None
```

Comment:
Do we still need this? Can we remove it? For `LightGBMModelAssembler` and `XGBoostModelAssembler` classes as well.